backfiller/backfiller/main.py

```diff
@@ -26,6 +28,29 @@
 	['remote', 'channel', 'quality', 'hour'],
 )
 
+hours_backfilled = prom.Counter(
+	'hours_backfilled',
+	'Number of hours successfully backfilled',
+	['remote', 'channel', 'quality'],
+)
+
+hash_mismatches = prom.Counter(
+	'hash_mismatches',
+	'Number of segments with hash mismatches',
+	['remote', 'channel', 'quality', 'hour'],
+)
+
+get_node_errors = prom.Counter(
+	'node_fetch_errors',
+	'Number of errors fetching a list of nodes',
+	['filename', 'database'],
+)
+
+backfill_errors = prom.Counter(
+	'backfill_errors',
+	'Number of errors backfilling',
+	['remote', 'channel'],
```

Comment:
no quality label?

Reply:
One backfill can do multiple quantities.

Reply:
I can easily move where the counter is so it counts each quality as its own backfill.